src/library_egl.js

```diff
@@ -62,6 +62,9 @@ var LibraryEGL = {
           } else if (param == 0x3032 /*EGL_SAMPLE_BUFFERS*/) {
             var samples = {{{ makeGetValue('attribList', '4', 'i32') }}};
             EGL.contextAttributes.antialias = (samples == 1);
+          } else if (param == 0x3100 /*EGL_CONTEXT_PRIORITY_LEVEL_IMG*/) {
+            var requestedPriority = {{{ makeGetValue('attribList', '4', 'i32') }}};
+            EGL.contextAttributes.lowLatency = requestedPriority < 0x3103 /*EGL_CONTEXT_PRIORITY_LOW_IMG*/;
```

Comment:
this depends on the order of those constants. how about `!=` instead of `<`?

Reply:
Fixed in 573801c